gating/pre_merge_test/run_system_tests.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+## Deploy virualenv for testing enironment molecule/ansible-playbook/infratest
+
+## Shell Opts ----------------------------------------------------------------
+
+set -ev
+set -o pipefail
+
+## Variables -----------------------------------------------------------------
+
+RE_HOOK_ARTIFACT_DIR="${RE_HOOK_ARTIFACT_DIR:-/tmp/artifacts}"
+SYS_WORKING_DIR=$(mktemp  -d -t system_test_workingdir.XXXXXXXX)
+export SYS_VENV_NAME="${SYS_VENV_NAME:-venv-molecule}"
+SYS_TEST_SOURCE_BASE="${SYS_TEST_SOURCE_BASE:-https://github.com/rcbops}"
+SYS_TEST_SOURCE="${SYS_TEST_SOURCE:-rpc-openstack-system-tests}"
+SYS_TEST_SOURCE_REPO="${SYS_TEST_SOURCE_BASE}/${SYS_TEST_SOURCE}"
+SYS_TEST_BRANCH="${SYS_TEST_BRANCH:-master}"
```

Comment:
I noticed there's no `execute_tests.sh` on master branch of this repo, should this be pointing at `johnduarte-asc-144-create-molecule-core` branch instead?

Reply:
@mattt416 I would rather block the merge of this PR on the merge of https://github.com/rcbops/rpc-openstack-system-tests/pull/2 rather than point to the topic branch in this PR. My thought is that it is not a critical dependency and the run command will simply fail. I you disagree, I am happy to flag this PR as blocked until the other one is merged.

Reply:
I'm going to merge this @johnduarte , but leave the PR in rpc-gating pending your PR in rpc-openstack-system-tests.